pipeline/trusted-resources/cmd/sign/sign_test.go

```diff
@@ -178,9 +135,49 @@ func unmarshal(t *testing.T, buf []byte) (*v1beta1.TaskRun, []byte) {
 	if err != nil {
 		t.Fatalf("error decoding signature: %v", err)
 	}
+
+	delete(tr.ObjectMeta.Annotations, trustedtask.SignatureAnnotation)
```

Comment:
Why do we need to delete the annotation here? (wouldn't it already be removed prior to signing?)